components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1444,6 +1445,54 @@ void RewardsServiceImpl::SetConfirmationsIsReady(const bool is_ready) {
     ads_service->SetConfirmationsIsReady(is_ready);
 }
 
+void RewardsServiceImpl::GetAdsNotificationsHistory(
+    const uint64_t from_timestamp,
```

Comment:
`from_timestamp` should be renamed to `from_timestamp_seconds` as otherwise gets confusing with JS which is normally ms